### PR TITLE
refactor: remove unnecessary return await

### PR DESCRIPTION
**Pull request by clarkdo**

### CODE REVIEW

File: lib/app/utils.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ export function flatMapComponents(route, fn) {
 }
 
 export async function resolveRouteComponents(route) {
```
Comment (pi0) [inline lib/app/utils.js]: So now we can also remove async from `async function` :D

Comment (clarkdo) [reply lib/app/utils.js]: oh, that's right.

File: lib/app/server.js [hunk 2]
```diff
@@ -78,7 +78,7 @@ export default async ssrContext => {
   }
   const render404Page = async () => {
```
Comment (pi0) [inline lib/app/server.js]: This can be `() =>` without async because anyway we return a Promise

**Review by pi0 — APPROVED** [review]
Great!

**Pull request merged**